modules/op-release-notes-1-1.adoc

```diff
@@ -0,0 +1,103 @@
+* You can now add a Readiness Probe for EventListener Deployment at `URL/live`.
+* Support for embedding `TriggerBinding` specifications in EventListener triggers.
+* Trigger resources are now annotated with the recommended `app.kubernetes.io` labels.
+
+== Deprecated features
+The following items are deprecated in this release:
+
+* The `--namespace` or `-n` flags for all cluster-wide commands, including the `clustertask` and `clustertriggerbinding` commands, are deprecated. It will be removed in the next release.
+* The `name` field in `triggers.bindings` within an EventListener has been deprecated in favor of the `ref` field and will be removed in a future release.
+* Variable interpolation in TriggerTemplates using ``$(params)` has been deprecated in favor of using `$(tt.params)` to reduce confusion with the Pipeline variable interpolation syntax. The `$(params.<paramName>)` syntax will be removed in the next release.
+* The `tekton.dev/task` label is deprecated on `ClusterTasks`.
+* The `TaskRun.Status.ResourceResults.ResourceRef` field is deprecated and will be removed.
+* The `tkn create -f` command has been deprecated.
+* Namespace validation has been removed from `tkn` commands.
+* The default timeout of `1h` and the  `-t` flag for the `tkn ct start` command have been removed.
+* The `s2i ClusterTask` has been deprecated.
+
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+* Conditions do not support Workspaces
+* The `--workspace` option and the interactive mode is not supported for the `tkn clustertask start` command.
+* Support of backward compatibility for `$(params.<paramName>)` forces you to use trigger templates with pipeline specific params as the triggers webhook is unable to differentiate trigger params from pipelines params.
+* Pipeline metrics report incorrect values when you run a  promQL query for `tekton_taskrun_count`  and `tekton_taskrun_duration_seconds_count`.
+* PipelineRuns and TaskRuns continue to be in the `Running` and `Running(Pending)` states respectively even when a non existing PVC name is given to a Workspace.
+
+[id="fixed-issues-1-0_{context}"]
+== Fixed issues
+* Previously, the `tkn task delete <name> --trs` command would delete both the Task and ClusterTask if the name of the Task and CusterTask were the same. With this fix, the command deletes only the TaskRuns that are created by the `task <name>`.
+* Previously the  `tkn pr delete -p <name> --keep 2` command would disregard the `-p` flag when used with the `--keep` flag and would delete all the PipelineRuns except the latest two. With this fix, the command deletes only the PipelineRuns that are created by the `Pipeline <name>`, except for the latest two.
+* The `trigger triggertemplate describe` output now displays ResourceTemplates in a table format instead of YAML format.
```

Comment:
@khrm `Error: unknown command "trigger" for "tkn"`... what is the correct string here? `tkn triggertemplate describe`?

Reply:
Yes, it should be `tkn triggertemplate describe`.